src/pages/Contributors/styles.js

```diff
@@ -81,7 +81,10 @@ export const useStyle = makeStyles((theme) => ({
       textTransform: 'none',
     },
     '&$tabSelected': {
-      color: '#006B95',
+      color: '#0D99C6',
+      borderStyle: 'none none solid none',
+      borderColor: '#0D99C6',
+      borderWidth: '5px',
```

Comment:
This styling is not quite right for the line underneath each tab. I see two colors.

It is also not part of the issue. Meaning, it's not a font thing. It's fine to sneak things in, but it's also good to keep work focused. 

Please remove the changes to `tabSelected`. We'll make a separate issue for tab styling.

Otherwise, I will merge this PR! Thank you.

Reply:
completed! updated the PR, thanks